app/Controller/CoGroupsController.php

```diff
@@ -681,6 +792,64 @@ public function paginationConditions() {
       $pagcond['conditions']['CoGroup.group_type'] = $searchterm;
     }
 
+    // Exclude admin groups
+    if(!empty($this->params['named']['search.noadmin'])) {
+      $pagcond['conditions']['CoGroup.group_type <>'] = 'A';
```

Comment:
Use enums, not values.

```suggestion
      $pagcond['conditions']['CoGroup.group_type <>'] = GroupEnum::Admins;
```

Reply:
Fix committed